hydra-cluster/src/HydraNode.hs

```diff
@@ -215,11 +214,12 @@ withHydraCluster ::
   [SigningKey HydraKey] ->
   -- | Transaction id at which Hydra scripts should have been published.
   TxId ->
+  ProtocolParameters ->
```

Comment:
This list of parameters is starting to get quite large and unwieldy, perhaps an opportunity for more refactoring later?

Reply:
Yes definitely